nominal/core/channel.py

```diff
@@ -114,6 +117,80 @@ def _from_conjure_logicalseries_api(
             _clients=clients,
         )
 
+    def get_decimated(
+        self,
+        start: str | datetime | IntegralNanosecondsUTC,
+        end: str | datetime | IntegralNanosecondsUTC,
+        *,
+        buckets: int | None = None,
+        resolution: int | None = None,
+    ) -> pd.DataFrame:
+        """Retrieve the channel data as a pandas.Dataframe, decimated to the given buckets or resolution.
+
+        Enter either the number of buckets or the resolution for the output.
```

Comment:
is it valid to provide both? if not, lets check before we make the API call?

Reply:
Providing both results in an API error.